frontend/src/common/images/logo.svg

```diff
@@ -0,0 +1,4 @@
 2.8-20.5 2.8-10.8 0-13.3-.3-18.5-2.3-13-5.1-21.9-20.1-22.7-38.6l-.3-6 57.3-.3 57.4-.2-.5-26.8c-.3-21.8-.8-28.5-2.5-36.7-6.9-33.5-22.2-54.5-46.7-64.2-14.3-5.7-36.2-7.7-52.3-4.7zm25.5 47.8c7.4 2 10.8 4.8 14.3 11.9 3.4 7 5 14.9 5 25.4v7.3h-57v-5.3c.1-10.9 4.7-26.4 9.7-32.3 6.3-7.3 16.9-10 28-7zm544.8-47.9c-13.8 2.6-23.4 7.5-33.7 17.3-16.9 16.1-24.5 37.9-26.9 77.8-2 32 1 63.7 7.7 82.7 11.8 33 35.5 50.7 68.1 50.7 12.5 0 25.5-5.4 35.4-14.7 2.5-2.4 4.9-4.3 5.3-4.3.3 0 .6 6 .6 13.2 0 27.2-6 36.8-25.7 41.4-14.7 3.5-37.9-2.9-52.1-14.4-2.3-1.9-4.5-3.3-4.7-3-1.5 1.4-18.5 37.2-18.5 38.8 0 4 13.8 12.9 30 19.4 3.6 1.5 11.2 3.6 17 4.7 14.7 3 38.1 2.3 52-1.4 33.9-9 52.5-29.3 58.9-64.3.8-4.1 1.1-42.5 1.1-123.7V166h-25.4c-14 0-25.7.3-26 .8-.2.4-.9 4.3-1.5 8.6-.6 4.4-1.3 8.2-1.6 8.4-.2.2-3.3-2.5-7-6.1-13-13.1-31.9-18.1-53-14.2zm37.5 48c5.6.9 11.8 4.3 15.9 9l3.1 3.6v53.7c-.1 60.5.7 54.7-8.3 61.2-4.5 3.2-5.4 3.4-14.1 3.8-11.9.5-18.2-1.3-24-7.2-8-8-11.1-18.2-12.8-42.8-2-28.3 1.2-56.3 7.7-67.4 4.1-7.1 10.5-12 17.1-13.4l6.9-1.5c.6-.2 1.7-.2 2.5 0s3.5.7 6 1zm185.5-47.9c-40 7.1-63.3 35.4-69.6 84.4-1.5 12-1.5 50.6 0 62.5 1.3 10.2 5 23.9 8.6 32 3.9 8.9 9.3 16.5 17.5 24.6 12 11.8 22.9 17.6 41.5 22.2 7.5 1.8 11.8 2.2 26 2.2 15.4-.1 17.9-.3 27-2.8 18.5-5.1 36.3-15.4 44.5-25.7l3.2-4.1-2.5-3.7-11-16.9c-4.8-7.3-8.8-13.3-9.1-13.3s-3.5 2.3-7.1 5c-13.5 10.2-23.2 13.5-40.5 13.4-14.9 0-21.3-2-28.2-8.7-7.2-7.1-12.8-22-12.8-34.3V296h115v-13.3c-.1-52.2-7.5-79.4-27.5-99.7-16.3-16.7-46.5-24.5-75-19.4zm30.3 49.3c9.1 4.7 13.3 14.4 14.8 34.4l.7 8.7h-58.6l.7-7.3c2.6-27.5 12.5-39.6 31.4-38.2 4.2.3 8.8 1.3 11 2.4zm195.7-49c-3.8 1-8.8 2.6-11 3.6-5.8 2.5-15.3 9.5-20.5 15.1l-4.5 4.8-.8-9.4c-.4-5.2-.9-10.1-1.2-10.8-.3-1-6.3-1.2-27.2-1l-26.8.3-.3 110.8-.2 110.7 29.2-.2 29.3-.3.5-161.7 3.5-4.8c9.4-12.8 32.2-13.7 40.4-1.5 5.4 7.9 5.4 8.8 5.8 91.8l.4 76.7 28.7-.2 28.7-.3.3-69.5-.8-85c-2.3-34.1-12.8-55.7-31.8-65.4-10.5-5.4-28.8-7-41.7-3.7zm180 .1c-16.2 3.3-27.3 8.6-37.8 18.1-13 11.8-21.2 26.2-26.5 46.9-3.9 15.1-4.7 23.6-4.7 51.5.1 24 .3 28.2 2.3 37.7 5.5 26.6 16.1 44 34.9 57.4 23.2 16.7 61.9 21.1 92.2 10.7 16.4-5.6 28.2-13.2 37.9-24.4l2.2-2.6-10.6-16.4-11.4-17.2c-.4-.5-4.1 1.6-8.1 4.7-12.7 9.7-28.3 14.4-44.4 13.3-16-1-25.2-6.7-31.1-19.3-3.3-7.1-7.1-24.1-6-27 .5-1.2 9.6-1.4 57.7-1.4h57.1l-.4-26.8c-.6-32.6-2-41.4-10-61.2-7.9-19.6-23-34.3-41.7-40.6-16.2-5.4-35.4-6.7-51.6-3.4zm32.1 49.1c8.6 4.1 13.3 14.7 14.1 31.6l.6 11.3H2016v-4.2c0-6.7 3-20.9 5.6-26.3 5-10.8 13.7-15.9 25.7-15.1 4.2.3 8.6 1.4 11.3 2.7zm-879.2 5.6l-9.8 10.5c-1 .7-.9 1 6.9 10.4l13.8 16.8 15 18.2c6.4 7.8 11.7 14.5 11.7 14.9 0 .3-6 7.8-13.2 16.7l-33.8 42.2c.1.6 4.3 5.6 9.5 11.1 8.2 8.8 9.6 9.8 10.8 8.5 1.6-1.7 26.6-32 37.6-45.8 4.1-5 7.8-9.2 8.3-9.2.4 0 4 4 8 8.8l23.2 28c8.7 10.6 16.2 19.1 16.7 19 1.1-.4 18.9-19.8 18.9-20.6 0-.4-10.4-13.3-23.1-28.7l-23.9-29c-.6-.7 8.5-12.4 23.4-30.5l24.4-29.5-2.9-3.5c-5.4-6.6-16.7-18.5-17.1-18-.3.3-11 13.3-23.8 28.9l-23.4 28.4-3.4-3.9c-4.1-4.7-36.5-44.1-40.7-49.5-1.6-2.1-3.3-3.8-3.6-3.9-.4 0-4.6 4.4-9.5 9.7z"/>
```

Comment:
Do we have defined use cases for black vs pink versions of the logo?

Reply:
I don't think so! @pdugan20 is indifferent between either, and so I picked black for the homepage to match the cellxgene page!

What's your thought on this 😄 ?

Reply:
No thoughts, just curious on if we have a standard I should be following in the future.  For now, it seems that the only use cases for the colored logo are print and publishing (stickers, features in papers/posters, our doc website)

Reply:
Ah awesome context to keep in mind! Thanks for sharing that, @seve 🙏 👌